packages/babel-plugin-proposal-private-methods/test/fixtures/private-static-method/read-only/output.js

```diff
@@ -0,0 +1,9 @@
+class A {
+  run() {
+    babelHelpers.classStaticPrivateMethodSet(A, A, _method, 2);
```

Comment:
In a followup, we need to make this better. We're passing `_method` where we expect `descriptor`.

Reply:
In Babel 8 I'd like to just remove the concept of descriptors for private elements.
I introduced them for decorators, but the decorators proposal has evolved to be more static and now we don't need runtime descriptors anymore.